CRM/Export/BAO/Export.php

```diff
@@ -157,7 +157,7 @@ public static function getGroupBy($processor, $returnProperties, $query) {
       $groupBy = "civicrm_activity.id ";
     }
 
-    return $groupBy ? ' GROUP BY ' . $groupBy : '';
+    return $groupBy ? ' GROUP BY ' . implode(', ', (array) $groupBy) : '';
```

Comment:
@eileenmcnaughton can we set $groupBy = NULL at L127 and then:

```php
return !$groupBy ?  '' : ' GROUP BY ' . implode(', ', (array) $groupBy);
```

Reply:
@monishdb I don't think that is any different? I just changed it to null since it seems a little cleaner but it still evaluates to FALSE either way.

The return line is just the same thing re-ordered?